Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -312,10 +312,47 @@ extension SPNoteEditorViewController: SPCardPresentationControllerDelegate {
 //
 extension SPNoteEditorViewController {
 
-    private func presentInformationController(for note: Note) {
-        let information = NoteInformationViewController(note: note)
-        information.configureToPresentAsCard()
-        present(information, animated: true, completion: nil)
+    /// Present information controller
+    /// - Parameters:
+    ///     - note: Note
+    ///     - barButtonItem: Bar button item to be used as a popover target
+    ///
+    @objc
+    func presentInformationController(for note: Note, from barButtonItem: UIBarButtonItem) {
+        let informationViewController = NoteInformationViewController(note: note)
+
+        let presentAsPopover = UIDevice.sp_isPad() && traitCollection.horizontalSizeClass == .regular
+
+        if presentAsPopover {
+            let navigationController = SPNavigationController(rootViewController: informationViewController)
+            navigationController.configureAsPopover(barButtonItem: barButtonItem)
+            navigationController.displaysBlurEffect = true
+            self.informationViewController = navigationController
+            present(navigationController, animated: true, completion: nil)
+        } else {
+            informationViewController.configureToPresentAsCard()
+            self.informationViewController = informationViewController
+            present(informationViewController, animated: true, completion: nil)
+        }
+    }
+
+    /// Dismiss and present information controller.
+    /// Called when horizontal size class changes
+    ///
+    @objc
+    func updateInformationControllerPresentation() {
+        guard let informationViewController = self.informationViewController else {
+            return
+        }
```

Comment:
I was gonna suggest this:

```
        guard let informationViewController = presentedViewController as? NoteInformationViewController else {
            return
        }
```

But the `presentedViewController` is actually a UINavigationController instance. And that breaks my legs! :joy:

Reply:
Having the VC presented using both card and popover is not easy 😄